rpm/0016-Enable-external-window-usage.patch

```diff
@@ -94,11 +113,64 @@ index c12d381..781dcfb 100644
    if (!gl() || gl()->IsDestroyed())
      return false;
  
++  if (gfxPrefs::UseExternalWindow()) {
```

Comment:
This must be true over here as there is a return already earlier if gfxPrefs::UseExternalWindow was false.